virttest/qemu_devices/qcontainer.py

```diff
@@ -1812,8 +1812,8 @@ def memory_define_by_params(self, params, name):
         pc-dimm devices.
         """
         devices = []
-        if not self.has_option("object"):
-            logging.warn("QOM does not support by your qemu")
+        if not self.has_device("pc-dimm"):
+            logging.warn("'PC-DIMM' does not support by your qemu")
```

Comment:
`self.has_device` returns `True` when `pc-dimm` device is available. Should remove `not` to match with the `logging.warn` message.
`if self.has_device("pc-dimm"):`
          ` logging.warn("'PC-DIMM' does not support by your qemu")`